src/utils/ShareButton/shareTracking.js

```diff
@@ -0,0 +1,10 @@
+const shareTracking = channel => {
+  const dataLayer = window.dataLayer || [];
+
+  dataLayer.push({
+    event: 'btn_social-share',
+    action: channel
```

Comment:
Would it be useful to record the URL as well?

Reply:
@phiral how about we record `url` as well?

Reply:
@gusliedke no need for the URL, I can grab that from the history change event.

Reply:
When I send the info in Google Analytics I send the page URL with the event. So no need to specify the url.